Articles/Blog/2023-03-08-productivity-practices-and-tools.md

```diff
@@ -0,0 +1,155 @@
+#### Key Journaling Strategies
+
+What to journal and how to journal are deeply personal.
+Here are a few of the high level strategies to consider:
+
+First, treat a journal as an append only log.
+Journaling is most useful when one can make entries quickly and easily.
+The more effort involved, the less likely it will be done.
+Now, one may ask, *"if the journal is append only, and everything is jumbled together, how can you ever find anything?"*
+The answer is in the second key strategy.
+
+Second, periodically reflect and distill what was journaled.
+Sort through the material to organize what is truly important and copy into more permanent homes.
+Bullet journaling recommends using pen and paper journals, because it requires effort to physically copy things from one place to another.
+
+Common reflection cadences are daily, weekly and monthly.
+Where appropriate, it might be worthwhile to align reflection cadence with a project's sprint cadence.
+[Building a Second Brain](https://www.buildingasecondbrain.com/) recommends *progressive summarization* which involves underlining important entries upon daily review, highlighting important underlined entries upon weekly review and migrating (copying) important highlighted entries upon monthly review.
+
+At longer cadences, more substantial reflection and review is appropriate.
+For example, at monthly intervals, its reasonable to check in with yourself and ask...what should I stop, start, and continue for next month?
+Are my practices productive and effective or do I need to make some adjustments?
+At quarterly intervals, ask yourself...what is my vision/theme/aspiration(s) for this season?
+How well did I live up to my aspirations last quarter?
+
+#### Tools to consider
+
+There is a video series from [Tiago Forte](https://www.youtube.com/playlist?list=PLVNXAaej57W7fbYoc_XJ0bA0T3k_Ye-Q2) that describes a variety of journaling tools.
+In addition, consider [Drafts](https://getdrafts.com/), [GoodNotes](https://www.goodnotes.com/), [Notion](https://www.notion.so/), [Obsidian](https://obsidian.md/), [Evernote](https://evernote.com/).
+
+### Task Management
+
+While journaling aims to help *you* to be a generally more productive person, task management enables productivity of a particular plan of action.
+If tasks are not managed, they can wind up being left undone or, maybe worse, done without respect for priorities.
+In turn, this can hurt longer term goals.
+
+For many, the tactile act of checking tasks off as done or moving tasks across a project board (e.g. Kanban) has great appeal because it provides a sense of progress.
+
+#### What goes in to task management?
+
+When defining tasks, make them SMART -- specific, measurable, attainable, relevant, and timely.
+
+* Specific -- Specify as many of the [Five W's](https://en.wikipedia.org/wiki/Five_Ws) as possible (Who is responsible? What is the definition of done? When does it need to be completed? Where does it need to happen? Why is it necessary?)
```

Comment:
The definition of done in my mind falls under "measurable", and some of the 5 w's also fall under other categories such as timely.  Additionally, some would push against including the 5 w's here because you don't need to have verbose entries for everything in your task manager for personal use that would answer these questions.  For example, I have an entry that just says `cusz tests` and I know what that means and who I need to talk to get it done. That is why I prefer the phrase unambigious here.

Reply:
Agreed...there is surely some overlap among the SMART categorizations. I think everyone would agree that a *deadline* is really part of being **Specific**. I don't some overlap really hurts though. I've changed `possible` to `practical`. That way, if some are not practical as in the `cusz test` example...thats fine.

Fixed in 8865e0378f317026612320bd09de46e71e9fbfc9

Reply:
I accept this with the rephrasing to "practical"